components/actionbar/definitions.css

```diff
@@ -0,0 +1,55 @@
+/* stylelint-disable spectrum-tools/no-unused-custom-properties -- definitions file */
+
+/*!
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-ActionBar {
+	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-100);
+
+	/* item counter field label */
+	--spectrum-actionbar-item-counter-font-size: var(--spectrum-font-size-100);
+	--spectrum-actionbar-item-counter-line-height: var(--spectrum-line-height-100);
+
+	/* colors - applied to popover */
+	--spectrum-actionbar-popover-background-color: var(--spectrum-gray-25);
+	--spectrum-actionbar-popover-border-color: var(--spectrum-gray-400);
+
+	/* emphasized variation colors */
+	--spectrum-actionbar-emphasized-item-counter-color: var(--spectrum-white);
+
+	/* spacing of action bar bottom and horizontal outer edge */
+	--spectrum-actionbar-spacing-outer-edge: var(--spectrum-spacing-300);
+
+	/* spacing of close button */
+	--spectrum-actionbar-spacing-close-button-top: var(--spectrum-spacing-100);
+	--spectrum-actionbar-spacing-close-button-start: var(--spectrum-spacing-100);
+	--spectrum-actionbar-spacing-close-button-end: var(--spectrum-spacing-75);
+
+	/* spacing of item counter field label */
+	--spectrum-actionbar-spacing-item-counter-end: var(--spectrum-spacing-400);
+
+	/* spacing of action group */
+	--spectrum-actionbar-spacing-action-group-top: var(--spectrum-spacing-100);
+	--spectrum-actionbar-spacing-action-group-end: var(--spectrum-spacing-100);
+
+	/* drop shadow */
+	--spectrum-actionbar-shadow-horizontal: var(--spectrum-drop-shadow-x);
+	--spectrum-actionbar-shadow-vertical: var(--spectrum-drop-shadow-y);
+	--spectrum-actionbar-shadow-blur: var(--spectrum-drop-shadow-blur);
+	--spectrum-actionbar-shadow-color: var(--spectrum-drop-shadow-color);
+
+	/* cjk language support for item counter */
+	&:lang(ja),
```

Comment:
I could go either way on gating the custom property definition(s) here behind the `lang`—we could or could not do this—but if we decide to use `:root` instead of the class name selector, I guess it won't matter.

Reply:
Yeah I don't think the definition needs to be gated, just the application of it